server/command.go

```diff
@@ -236,3 +237,34 @@ func (p *Plugin) runPopCommand(args []string, extra *model.CommandArgs) (bool, e
 
 	return false, nil
 }
+
+func getAutocompleteData() *model.AutocompleteData {
+	todo := model.NewAutocompleteData("todo", "[command]", "Avaliable commands: list, add")
+
+	add := model.NewAutocompleteData("add", "[message]", "Adds a Todo")
+	add.AddTextArgument("Add a todo", "[message]", "")
+	todo.AddCommand(add)
+
+	list := model.NewAutocompleteData("list", "[name]", "Lists your Todo issues")
+	items := []model.AutocompleteListItem{{
+		HelpText: "Received Todos",
+		Item:     "in",
+	}, {
+		HelpText: "Sent Todos",
+		Item:     "out",
+	}}
```

Comment:
From looking a the the options it seams like only `/todo list in` and `/todo list out` are valid commands. But `/todo list` is also one. The current data model doesn't allow `""` as `Item`, but maybe it's helpful to add this to ensure optional arguments can be made clear e.g.:
![Screenshot from 2020-05-28 11-55-03](https://user-images.githubusercontent.com/16541325/83127127-155db980-a0da-11ea-834a-b93381d401d3.png)

Reply:
I talked about this situation with @asaadmahmood, he suggested that user should press escape if they want to end autocomplete earlier, that is why you can not have empty suggestion by design. Probably better UX here is to add `in` and `out` as optional parameters.

Reply:
It doesn't seam obvious to me that escape should be used. The dialog "forces" me as a user to pick either of these two.

![Screenshot from 2020-06-08 13-09-05](https://user-images.githubusercontent.com/16541325/84023927-4024f900-a989-11ea-84ee-b8d7febe4cd9.png)

@asaadmahmood Thoughts on this discussion?

Reply:
True, you have to press `esc`

Reply:
@DHaussermann Followed up with a ticket https://mattermost.atlassian.net/browse/MM-26282